addons/mail/static/src/core/web/follower_subtype_dialog.js

```diff
@@ -48,7 +48,6 @@ export class FollowerSubtypeDialog extends Component {
 
     async onClickApply() {
         const selectedSubtypes = this.state.subtypes.filter((s) => s.followed);
-        const thread = this.props.follower.thread;
```

Comment:
Could also have kept the const and reuse it in the 2 occurrences of `this.props.follower.thread` below.
With eslint-prettier applied, this would result in 3 less LOCs.

Reply:
This is to me a linter issue.
I do enjoy knowing where variables come from. Aliasing to a `const` feels practical at the moment but obfuscates future reading.